samples/client/petstore/java/resttemplate-jakarta/build.gradle

```diff
@@ -114,7 +114,7 @@ dependencies {
     implementation "com.fasterxml.jackson.core:jackson-core:$jackson_version"
     implementation "com.fasterxml.jackson.core:jackson-annotations:$jackson_version"
     implementation "com.fasterxml.jackson.core:jackson-databind:$jackson_databind_version"
-    implementation "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:$jackson_version"
+    implementation "com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-json-provider:$jackson_version"
```

Comment:
thanks for the PR and updating the samples as part of the PR.

in the latest master (not this branch), if you run `gradle build` in `samples/client/petstore/java/resttemplate-jakarta`, did you get any errors? it builds fine for me so wondering how I can repeat the issue to confirm the fix.